lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -907,6 +907,28 @@ def test_check_quality_control_conventions_for_quality_control_variable(self):
         self.assertEqual(len(passed_var), len(good_var))
         self.assertEqual(set(passed_var), set(good_var))
 
+    def test_check_quality_control_global(self):
+        self.imos.setup(self.new_dataset)
+        ret_val = self.imos.check_quality_control_global(self.new_dataset)
+        self.assertEqual(len(ret_val), 4)
+        passed_var = [r.name[1] for r in ret_val if r.value]
+        good_var = ['DEPTH_quality_control', 'TEMP_quality_control']
+        self.assertEqual(len(passed_var), len(good_var) * 2)
+        self.assertEqual(set(passed_var), set(good_var))
+
+        self.imos.setup(self.test_variable_dataset)
```

Comment:
For clarity it would be great to have variable names for test files that match the actual cdl file names:

test_variable_dataset == imos_test_variable.cdl ?